draft-ietf-oauth-transaction-tokens.md

```diff
@@ -542,7 +565,7 @@ Txn-Tokens SHOULD NOT be logged if they contain Personally Identifiable Informat
 
 # IANA Considerations {#IANA}
 
-This specification registers the following claims defined in Section {{txn-token-header}} to the OAuth Access Token Types Registry defined in {{RFC6749}}, and the following claims defined in Section {{txn-token-claims}} in the IANA JSON Web Token Claims Registry defined in {{RFC7519}}
+This specification registers the following claims defined in Section {{txn-token-header}} to the OAuth Access Token Types Registry defined in {{RFC6749}}, the following claim in Section {{subject-token-types}} to the "OAuth URI" subregistry of the "OAuth Parameters" {{IANA.OAuth.Parameters}} registry, and the following claims defined in Section {{txn-token-claims}} in the IANA JSON Web Token Claims Registry defined in {{RFC7519}}
```

Comment:
most of these things are not claims

Reply:
I realize much of this text is preexisting but it'd be good if the PR didn't introduce more misstatements

Reply:
Created separate PR for tracking this: https://github.com/oauth-wg/oauth-transaction-tokens/pull/92

Reply:
#92 kinda misses the mark https://github.com/oauth-wg/oauth-transaction-tokens/pull/92#discussion_r1578619029 and the new text here is still problematic

Reply:
I will address the broader issues with the nomenclature (header, media-type, etc.) in a separate PR (#92). For this PR, I have addressed the language regarding the token type identifier. @bc-pi  Please review if that part is OK.